homeassistant/components/media_player/cast.py

```diff
@@ -494,6 +505,23 @@ def name(self):
         """Return the name of the device."""
         return self._cast_info.friendly_name
 
+    @property
+    def device_info(self):
+        """Return information about the device."""
+        cast_info = self._cast_info
+
+        if cast_info.model_name == "Google Cast Group":
+            return None
+
+        return {
+            'name': cast_info.friendly_name,
+            'identifiers': [
```

Comment:
Identifiers should be set with tuples

Reply:
Tuples with 1 entry are very error prone, as the comma is what makes it the tuple. We can pass in list because `DeviceEntry` will convert it to a set anyway.

Reply:
Ok, then perhaps I should update deconz to do the same?

Reply:
Made them sets. You're right that it's better to stay in line with how they will end up on the device.